src/components/state.tsx

```diff
@@ -0,0 +1,31 @@
+import { createContext, ReactNode, useState } from 'react';
+import { Person } from '../types';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const MyContext = createContext<any>(null);
+
+interface MyProvideProps {
+  children: ReactNode;
+}
+
+export const MyProvider = ({ children }: MyProvideProps) => {
+  const [person, setPerson] = useState<Person[]>([]);
```

Comment:
The initial state for `person` should be typed as `Person[]` for consistency and clarity, even though TypeScript can infer the type from the useState hook.